extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeChannelLinkHandlerFactory.java

```diff
@@ -31,7 +32,7 @@ public String getUrl(String id, List<String> contentFilters, String searchFilter
     @Override
```

Comment:
Please also handle the API URLs in `getId()` here

Reply:
Fixed while rebasing with the latest changes about accounts and channels